deepchem/feat/tests/test_huggingface_featurizer.py

```diff
@@ -0,0 +1,18 @@
+from deepchem.feat.huggingface_featurizer import HuggingFaceFeaturizer
+
+
+def testHuggingFaceFeaturizer():
+    # NOTE: The test depends on the sanity of the pretrained tokenizer,
```

Comment:
Can you explain more what the comment means? That the sanity of the pretrained tokenizer isn't guaranteed?

Reply:
The download took ~5 second.

By sanity, if the vocabulary is modified or deleted (which can be since we are depending on an external resource), the test might fail.

Reply:
Can you update the comment to have the explanation you gave? sanity is an ambiguous term while your explanation here is clearer

Reply:
updated comment in 607baf5